docs/pages/desktop-access/troubleshooting.mdx

```diff
@@ -110,6 +110,23 @@ Ensure that your firewalls allow inbound DNS traffic on port `53` from the
 instance(s) running Teleport's Windows Desktop Service to the LDAP server
 (Active Directory Domain Controller).
 
+### RDP Connection Failed
+
+You click `CONNECT` on a Windows host from the Teleport UI, a new tab opens but
+nothing is displayed other than the top bar and an error is displayed about a
+failed RDP connection. You may also see errors similar to:
+
+```text
+Rdp(Io(Os { code: 54, kind: ConnectionReset, message: "Connection reset by peer" }))
+```
+
+**Solution:** Configure Certificate for RDP Connection
+
+This means that your Domain Controller do not support secure cipher suites for TLS
```

Comment:
I don't know to be honest with you. I might have gaps in my knowledge of Domain Controllers and Windows Servers. In my understanding Domain Controllers handle authentication and enforces policies for other Windows Server connected to it. So I thought that if we configure Policy on a Domain Controller, then it will propagate it to the other Window Servers and enforce it. I might be wrong, and if that's the case, then I think that we'll have to do configuration steps on every Windows Server and not just Domain Controller.

Reply:
I've created another PR: https://github.com/gravitational/teleport/pull/14366 that will enable certificate auto-enrollment for clients. When the DC is configured in this way then individual desktops will use the right certificate so there is no need to configure it on every Windows Server that we want to connect to.